templates/base.html

```diff
@@ -121,7 +123,9 @@ <h1><a href="{{ SITEURL }}">{{ SITETITLE }}</a></h1>
 
       <ul class="social">
         {% for name, link in SOCIAL %}
-        <li><a class="sc-{{ name }}" href="{{ link }}" target="_blank"><i class="fa fa-{{ name }}"></i></a></li>
+          <li><a class="sc-{{ name }}" href="{{ link }}" target="_blank">
+            <i class="fa{% if name in ['envelope-open', 'rss'] %}s{% else %}b{% endif %} fa-{{ name }}"></i>
```

Comment:
If you want I can split this into more verbose else/if with several names. The problem is caused by the introduction of font "families" in Font Awesome 5: `envelope-open` and `rss` need `fas`, and all the other social media `fab` class.

Reply:
my suggestion it just to use `fas` and `fab` inside if else, it's more readable.